pdsClient/pdsClient.go

```diff
@@ -426,6 +426,45 @@ func (c *E3dbPDSClient) DecryptRecord(ctx context.Context, record Record) (Recor
 	return record, err
 }
 
+// DecryptAK takes a record and an eak response, decrypts the membership key used for encrypting the record, and the public key
+// for the writer of the record. Then decrypts the record and returns data
+func (c *E3dbPDSClient) DecryptAK(ctx context.Context, record Record, akResponse *GetEAKResponse) (Record, error) {
```

Comment:
add group into the method name so its clear when this function would be used, and spell out AccessKey but you don't need to save on characters at the expense of readability
`DecryptGroupAcessKey`

Reply:
err, see how confusing this name is, I should have asked for

`DecryptGroupRecordWithGroupEncryptedAccessKey`

Reply:
Also `groupEncryptedAccessKey` vs `akResponse` (please ask yourself what meaning akResponse has to a random /first time reader of this method)